supertylerc/died
# Correct `resource_name` typo

Closes #25.

Changes:

* Renamed `resouce_name` method to the correct `resource_name`

Caveats: None

Test Plan:

* Verify build passes
* Check TravisCI
* Verify TravisCI has no errors
